File: PRAK403-2410817310009-Muhammad_Ghazi_Rakhmadi.py
a, b = map(int, input("Input: ").split())
output = []  

if a > b:
    i, j = a, b
    while i >= b and j <= a:
        output.append(f"{i} {j}")
        i -= 1
        j += 1
elif b > a:
    i, j = a, b
    while i <= b and j >= a:
        output.append(f"{i} {j}")
        i += 1
        j -= 1
print(" - ".join(output))

File: PRAK403-2410817310009-Muhammad_Ghazi_Rakhmadi.c
#include <stdio.h>

int main() {
    int a, b;

    printf("Input: ");
    scanf("%d %d", &a, &b);

    if (a > b) {
        for (int i = a, j = b; i >= b && j <= a; i--, j++) {
            printf("%d %d", i, j);
            if (i != b) { 
                printf(" - ");
            }
        }
    } 
    else if (b > a) {
        for (int i = a, j = b; i <= b && j >= a; i++, j--) {
            printf("%d %d", i, j);
            if (i != b) { 
                printf(" - ");
            }
        }
    }
}